client/src/components/Userpage.js

```diff
@@ -0,0 +1,243 @@
+  },
+  buttongray: {
+    color: "#808080",
+  },
+  email: {
+    color: "#9EA1BC",
+  },
+}));
+
+let invisibleCount = 0;
+export default function UserPage() {
+  const classes = useStyles();
+  const [exploreCards, setExploreCards] = React.useState([
+    {
+      location: "Oslo",
+      country: "Norway",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: false,
+    },
+    {
+      location: "Chichibu",
+      country: "Japan",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Copenhagen",
+      country: "Denmark",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Vancouver",
+      country: "Canada",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Stockholm",
+      country: "Sweden",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: false,
+    },
+  ]);
+
+  const listOfExploreCards = exploreCards.map((exploreCard) => {
+    if (exploreCard["liked"] === true) {
+      if (invisibleCount < 2) {
+        invisibleCount++;
+        return (
+          <Grid item xs={3} className={classes.exploreContainer}>
+            <ExploreCard
+              location={exploreCard.location}
+              country={exploreCard.country}
+              img={exploreCard.img}
+              onLike={() => {
+                exploreCard["liked"] = false;
+              }}
+            ></ExploreCard>
+          </Grid>
+        );
+      }
+
+      if (invisibleCount > 1) {
+        invisibleCount = 1;
+        return (
+          <React.Fragment>
+            <Grid item xs={3} className={classes.invisible}>
+              <ExploreCard location="Cancun" country="Mexico"></ExploreCard>
+            </Grid>
+            <Grid item xs={3} className={classes.invisible}>
+              <ExploreCard location="Cancun" country="Mexico"></ExploreCard>
+            </Grid>
+            <Grid item xs={3} className={classes.exploreContainer}>
+              <ExploreCard
+                location={exploreCard.location}
+                country={exploreCard.country}
+                img={exploreCard.img}
+                onLike={() => {
+                  exploreCard["liked"] = false;
+                }}
+              ></ExploreCard>
+            </Grid>
+          </React.Fragment>
+        );
+      }
+    }
+  });
+
+  return (
+    <Grid container spacing={0} className={classes.grid}>
+      <Grid container spacing={1} className={classes.leftQuadrant}>
+        <Grid item xs={12} className={classes.container}>
+          <paper className={classes.alignment}>
+            <Avatar />
+          </paper>
```

Comment:
`paper` is not an HTML tag as far as I could see, and the MaterialUI version is `Paper`, so I think this paper tag is not actually doing anything, and could easily just be another Grid, or be removed, and the className be applied to the parent?